docs/troubleshooting.md

```diff
@@ -16,6 +16,16 @@ It means you don't have a default SR set on the pool you are importing XOA on. T
 
 XOA uses HVM mode. If your physical host doesn't support virtualization extensions, XOA won't work. To check if your XenServer supports hardware assisted virtualization (HVM), you can enter this command in your host: `grep --color vmx /proc/cpuinfo`. If you don't have any result, it means XOA won't work on this hardware.
 
+## Set or recover XOA VM password
+
+If you need for some reason to recover your XOA VM password, this can be done by setting xoa user password via the XenStore of the VM. The following is to be ran on your XCP-ng host:
+```
+xe vm-param-set uuid=<UUID> xenstore-data:vm-data/system-account-xoa-password=<password>
+```
+Where UUID is the uuid of your XOA VM.
+
+Then you need to restart the VM.
+
```

Comment:
IMHO, we should clarify that this is the password for the `xoa` user that allows to connect via console or SSH.

Also, there is no longer a default password for this user, we should update the doc: https://github.com/vatesfr/xen-orchestra/blob/c5a769cb29b180c190040379af3709c45c3a114d/docs/installation.md#L23

Reply:
Oh ok i didn't understood that, so this should not be in  troubleshooting but in the main doc as how to set your XOA user password?

Reply:
Yes, that's a good idea :slightly_smiling_face: